app/src/main/java/de/westnordost/streetcomplete/quests/moped/AddMopedAccess.kt

```diff
@@ -0,0 +1,48 @@
+package de.westnordost.streetcomplete.quests.moped
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.quest.NoCountriesExcept
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddProhibitedForMoped : OsmFilterQuestType<AddMopedAccessAnswer>() {
+
+    override val elementFilter =
+        //only include separate cycleways,
+        // in case of a cycleway that is part of a road, mopeds are assumed to be allowed on the road
+        """
+        ways with (
+            highway = cycleway
+        )
+        and !moped
+        and (motor_vehicle != no or !motor_vehicle)
+        """
+    override val enabledInCountries = NoCountriesExcept("BE")
+    override val defaultDisabledMessage = R.string.default_disabled_msg_visible_sign_moped
+
+
```

Comment:
```suggestion
    override val elementFilter = """
        ways with (
            highway = cycleway
            or highway = path and bicycle = designated
            or highway = footway and bicycle = designated
        )
        and !moped
        and (motor_vehicle != no or !motor_vehicle)
    """
    override val enabledInCountries = NoCountriesExcept("BE")
    override val defaultDisabledMessage = R.string.default_disabled_msg_visible_sign_moped
```

Separately mapped cycleways can also be mapped like `highway = path` + `bicycle = designated` etc.

Removed the comment, because it doesn't make sense. We don't assume this here in the quest.